jobs/config.json

```diff
@@ -38,13 +38,90 @@
       "sig-node"
     ]
   },
+  "ci-cri-containerd-e2e-gce-alpha-api": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-alpha-api.env",
+      "--env-file=jobs/env/ci-cri-containerd-e2e-gce.env",
+      "--extract=ci/latest",
+      "--gcp-node-image=gci",
+      "--gcp-project=k8s-e2e-gce-alpha-api-access",
+      "--gcp-zone=us-central1-f",
+      "--ginkgo-parallel=30",
+      "--provider=gce",
+      "--test_args=--ginkgo.focus=\\[Feature:GCEAlphaFeature\\] --minStartupPods=8",
+      "--timeout=60m"
+    ],
+    "scenario": "kubernetes_e2e",
+    "sigOwners": [
+      "sig-node"
+    ]
+  },
+  "ci-cri-containerd-e2e-gce-device-plugin-gpu": {
```

Comment:
cc @mindprince

Reply:
LGTM.

In testgrid, can you add:
```
- name: ci-cri-containerd-e2e-gce-device-plugin-gpu
  gcs_prefix: kubernetes-jenkins/logs/ci-cri-containerd-e2e-gce-device-plugin-gpu
```
under `wg-resource-management` as well.

Reply:
Will do.

Reply:
Done.